### ISSUE TITLE
Recommendation to new order of learning paths

### ISSUE THREAD

**Issue by mct370770** [MicrosoftLearning/MS-102T00-Microsoft-365-Administrator-Essentials#55]
# Learning Path: 9
#### Step: 48

It is bad for the experience that the students are not actually able to complete the lab while on the course.

It would have much better to swap Learning Path 8 and 9.
Thereby you would have created the sensitivity labels day number 4 and were able to se and use them on day 5.
It is also the preferred order to do things in.

**Comment by tfrink** [comment]
We will be updating the course shortly following the next round of reviews. During the review, I will propose your recommendation to our team of Subject Matter Experts who help guide the course design.

**Comment by mct370770** [comment]
Hi, I have given this question a lot of thinking. 
I really like labs in many of these "learning paths".
The problem that I experiance is a lack of knowledge given by the labs in comparison to the time used in each lab. In other words - the students are in many labs using quite a lot of time - just to show or experience very few details.
I feel you are asking me for more broad solution - in this case I would work more closely with the lab providers and make preprepared labs where many of the prerequistes vere done already - and thereby taking a more direct approch the subject at hand for each learning path. This way there are not too many labs - they are just not using the partipants time correctly.

Example:
Create a policy and show it is created - 
Next - Open a precreated lab with the (correct) policy created and show the result of what the policy can do.

**Comment by tfrink** [comment]
I understand where you're coming from. However, with the approach you mention, the students would not learn how to create a policy, which I think is one of the major attractions of an admin course such as this. If we just create the policy for them and let them test it to see what it can do, they're not learning about all the options associated with creating those policies. We already tell them in the training what it can do, but the lab is an opportunity to actually create the object. I may be wrong, but my fear is that we would get a lot of push back from students complaining that they want to learn how to create policies. However, I will definitely bring this up in our next planning session for the course and see what our subject matter experts have to say. Thank you so much for your thoughtful analysis and suggestions. It's always great to hear from our MCTs on how we can improve our courses.

**Comment by mct370770** [comment]
Hi Tony, 
I as well totally understand the learning by doing part that would be lost in my approach.
What if the exercise let the students create a policy - and instead of waiting for this policy to be applied - they would instead go on using a policy already created and applied?

**Comment by tfrink** [comment]
See the updated Lab 9. We added functionality to have the students run a lab setup script on day 1 that created and published a sensitivity label and label policy that matched exactly the label and label policy they created in Lab 9, Task 2. We then added a Task 3 that has the students assign the pre-created label to a document, and then Task 4, that has them verify the pre-created label policy.  In this way, students can test out basically the same label and label policy they created, just using pre-created versions with a different name.
